source/blog/2016-04-11-ember-2-5-released.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Ember.js 2.4-LTS, 2.5, and 2.6 Beta Released
+author: Matthew Beale
+tags: Releases
+---
+
+Ember 2.4-LTS, the first Ember Long-term Support release, lands today as Ember
+2.4.5. Future versions of 2.4 will be considered part of the LTS channel.
+The LTS channel comes with a commitment
+that low-risk critical bugfixes and security patches will be backported to
+these versions. Additionally, any change to commonly used private APIs must be deprecated
```

Comment:
the word "our" is always a bit weird in these kinds of posts. I'll ponder some re-wording though.